lms/services/blackboard_api/_basic.py

```diff
@@ -106,9 +106,11 @@ def _send(self, method, url):
         try:
             return self._oauth_http_service.request(method, url)
         except ExternalRequestError as err:
+            err.refreshable = getattr(err.response, "status_code", None) == 401
```

Comment:
If refreshable it's only true when `status == 401 and message == "Bearer token is invalid"` maybe this err.refreshable line could be moved neaer the `raise`